packages/lib/src/index.ts

```diff
@@ -7,6 +7,25 @@ declare global {
   }
 }
 
+export enum MediaMimeType {
+  PLY = "application/x-ply",
+  TXT = "text/plain"
+}
+
+export enum MediaExtensions {
```

Comment:
is this used outside this file? not able to search for it. if not, we can simplify by removing it (e.g. just having mimeTypeToExtension map MediaMimeType to strings). (better to do that and only add this back when it's needed)

Reply:
changed to use string for now